components/picker/package.json

```diff
@@ -25,33 +25,41 @@
 	},
 	"main": "dist/index.css",
 	"peerDependencies": {
-		"@spectrum-css/helptext": ">=5",
-		"@spectrum-css/icon": ">=7",
-		"@spectrum-css/menu": ">=7",
-		"@spectrum-css/popover": ">=7",
-		"@spectrum-css/progresscircle": ">=3",
-		"@spectrum-css/tokens": ">=14 || >=15"
+		"@spectrum-css/helptext": ">=7.0.0 <8.0.0",
+		"@spectrum-css/icon": ">=9.0.0 <10.0.0",
+		"@spectrum-css/menu": ">=9.0.0 <10.0.0",
+		"@spectrum-css/popover": ">=8.0.0 <9.0.0",
+		"@spectrum-css/progresscircle": ">=5.0.0 <6.0.0",
+		"@spectrum-css/tokens": ">=16.0.0 <17.0.0"
 	},
 	"peerDependenciesMeta": {
 		"@spectrum-css/helptext": {
 			"optional": true
 		},
+		"@spectrum-css/icon": {
+			"optional": true
+		},
+		"@spectrum-css/menu": {
+			"optional": true
+		},
+		"@spectrum-css/popover": {
+			"optional": true
+		},
```

Comment:
These aren't optional for a functioning Picker. I'm seeing a lot of components with peer dependencies added as optional that aren't really optional, or is there some other criteria here for what is optional that I'm not aware of?